Query/ContractQuery.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Dta.Marketplace.Azure.Functions.Model;
+
+namespace Dta.Marketplace.Azure.Functions.Query {
+
+    internal class ContractQuery : BaseQuery {
+
+        private readonly string _query = @"
+select [Contract Supplier Marketplace SME Status], sum([Total Contracts Value]) [Total Contracts Value], sum([Total Contracts]) [Total Contracts]
+from (
+	select	
+		format ((select	MAX([Insert Timestamp])
+			from	[Data].[VW_RPT_AusTender_Contract] ac
+			where	ac.[Contract SON ID] in( 'SON3413842')),'yyyy-MM-dd') as [Last Updated],
```

Comment:
Do we need the second SON ID here?

Reply:
double checked and it wasn't in the original